mk/config.mk

```diff
@@ -858,15 +858,25 @@ CFG_CORE_ASYNC_NOTIF ?= n
 $(eval $(call cfg-enable-all-depends,CFG_MEMPOOL_REPORT_LAST_OFFSET, \
 	 CFG_WITH_STATS))
 
-# Pointer Authentication (part of ARMv8.3 Extensions) provides
-# instructions for signing and authenticating pointers against secret keys.
-# These can be used to mitigate ROP (Return oriented programming) attacks.
-# This option enables these instructions for TA's at EL0. When this option is
-# enabled , TEE core will initialize secret keys per TA.
-CFG_TA_PAUTH ?= n
+# Pointer Authentication (part of ARMv8.3 Extensions) provides instructions
+# for signing and authenticating pointers against secret keys. These can
+# be used to mitigate ROP (Return oriented programming) attacks.
```

Comment:
Practically speaking, and typically, this means having `paciasp`/`autiasp` at the beginning and end of functions, inserted by the compiler, in order to sign then verify ELR, doesn't it?
Assuming this is all there is to it at the moment, it would be nice to write it explicitly here I think.

Reply:
OK, I'll add something.